src/styles/utils/_mixin.scss

```diff
@@ -0,0 +1,24 @@
+@mixin hover($_property, $_toValue) {
+  transition: #{$_property} 0.3s;
+  &:hover {
+    #{$_property}: $_toValue;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width: 1280px) {
+    @content;
+  }
+}
+
+@mixin onTablet {
+  @media (min-width: 640px) and (max-width: 1279px) {
```

Comment:
Adjust the breakpoint values for better device coverage. Consider using 768px for min-width and 1024px for max-width for tablet devices.